apps/console/src/app/(protected)/policies-and-procedures/policies/create/page.tsx

```diff
@@ -0,0 +1,68 @@
+'use client'
+
+import React from 'react'
+import { PageHeading } from '@repo/ui/page-heading'
+import {
+  PoliciesForm,
+  PolicyFormSchema,
+} from '@/components/pages/protected/policies/policies-form'
+import { NextPage } from 'next'
+import { useCreateInternalPolicyMutation } from '@repo/codegen/src/schema'
+import { useRouter } from 'next/navigation'
+
+type PageProps = {}
+
+const Page: NextPage<PageProps> = () => {
+  const router = useRouter()
+
+  const [{ fetching: isSubmitting }, createPolicy] =
+    useCreateInternalPolicyMutation()
+
+  // const onSubmit = (data: z.infer<typeof PolicyFormSchema>) => {
+  //   updatePolicy({
+  //     name: data.name,
+  //     description: data.description,
+  //     purposeAndScope: data.purposeAndScope,
+  //     policyType: data.policyType,
+  //     background: data.background,
+  //   })
+  // }
+
+  const onSubmit = async (policy: PolicyFormSchema) => {
+    console.log('onSubmit: creating policy', policy)
+    const { data, error } = await createPolicy({
+      input: {
+        name: policy.name,
+        description: policy.description,
+        background: policy.background,
+        policyType: policy.policyType,
+        purposeAndScope: policy.purposeAndScope,
+        details: policy.details,
+      },
+    })
+    if (error) {
+      console.error('failed to save policy:', error)
```

Comment:
we should use `toast` to to the error to the user rather than logging it to the console